code/game/objects/items/mail.dm

```diff
@@ -191,6 +192,8 @@
 		goodies += this_job.mail_goodies
 		if(this_job.paycheck_department && department_colors[this_job.paycheck_department])
 			color = department_colors[this_job.paycheck_department]
+			if(!color)
```

Comment:
is there any case this is used?

Reply:
hmm... actually I think it won't be used. I simulated it a bit wrong. I will remove it.